workflow-templates/reuse.properties.json

```
@@ -0,0 +1,8 @@
+{
+  "name": "REUSE compliance check",
+  "description": "Check SPDX headers for REUSE compliance",
+  "iconName": "octicon codescan-checkmark",
```

```suggestion
  "iconName": "octicon verified",
```

https://primer.github.io/octicons/verified-24 matches better than https://primer.github.io/octicons/codescan-checkmark-24 ?